lib/ansible/modules/messaging/rabbitmq/rabbitmq_user.py

```diff
@@ -131,8 +140,6 @@ def __init__(self, module, username, password, tags, permissions,
         self.node = node
         if not tags:
             self.tags = list()
-        else:
-            self.tags = tags.split(',')
```

Comment:
This change appears to be responsible for the CI failure.

Reply:
I see, fixed that thinko.